driver/bpf/fillers.h

```diff
@@ -3156,7 +3266,7 @@ FILLER(sys_open_by_handle_at_x, true)
 {
 	/* Parameter 1: ret (type: PT_FD) */
 	long retval = bpf_syscall_get_retval(data->ctx);
-	int res = bpf_push_s64_to_ring(data, retval);
+	int res = bpf_val_to_ring(data, (s64)retval);
```

Comment:
Unfortunately using the upstream way causes verifier errors. We don't ever call this filler, so I think as long as we can satisfy the verifier, we should be ok. Having said that, it might be worth a comment here that describes why we can't use the new type-specific functions

Reply:
That's good to know. They specifically created this alternative function to deal with verifier errors so it sucks that it's doing the opposite, but validates my objection to it a couple months back.